pkg/resource/logging-secret/alloy-logging-secret.go

```diff
@@ -0,0 +1,106 @@
+package loggingsecret
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"text/template"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/Masterminds/sprig/v3"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	loggingcredentials "github.com/giantswarm/logging-operator/pkg/resource/logging-credentials"
+)
+
+//go:embed alloy/logging.alloy.template
+var alloyLogging string
+
+//go:embed alloy/logging-secret.yaml.template
+var alloyLoggingSecret string
+
+func GenerateAlloyLoggingSecret(lc loggedcluster.Interface, credentialsSecret *v1.Secret, lokiURL string) (v1.Secret, error) {
+	secret, err := generateAlloyLoggingConfig(lc, credentialsSecret, lokiURL)
```

Comment:
I'm a bit confused that we use a function named generateAlloyLoggingConfig in a function named generateAlloyLoggingSecret. Do we need that second function at all?

Reply:
I killed this function